ansible/roles/node/tasks/main.yml

```diff
@@ -2,3 +2,11 @@
 - name: Register NodeJS
   shell: 'curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -'
   changed_when: false
+  creates: '/etc/apt/sources.list.d/nodesource.list'
+
+- name: Install nodejs
+  apt:
+    name: nodejs
+    update_cache: yes
+    cache_valid_time: 3600
+  become: true
```

Comment:
Become shouldn't be needed here as it is defined in the playbook.

Reply:
Removed.